webapp/coverage/base.css

```
@@ -0,0 +1,224 @@
+body, html {
+  margin:0; padding: 0;
+  height: 100%;
+}
+body {
+    font-family: Helvetica Neue, Helvetica, Arial;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: Helvetica

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)